lms/djangoapps/grades/migrations/0005_multiple_course_flags.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import xmodule_django.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('grades', '0004_visibleblocks_course_id'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='coursepersistentgradesflag',
+            name='course_id',
+            field=xmodule_django.models.CourseKeyField(max_length=255, db_index=True),
+        ),
+    ]
```

Comment:
@jibsheet Are there any devops concerns with dropping an index in a migration? This table is empty on prod, and has 4 entries on stage.

```
edxapp@precise64:~/edx-platform$ ./manage.py lms --settings=devstack sqlmigrate grades 0005
BEGIN;
ALTER TABLE `grades_coursepersistentgradesflag` DROP INDEX `grades_coursepersistentgradesfla_course_id_539df2575c05593b_uniq`;

COMMIT;
```